lib/sidekiq/statistic/statistic/charts.rb

```diff
@@ -26,6 +21,10 @@ def color_for(worker)
           .join ','
       end
 
+      def rgb_to_hex(rgb)
```

Comment:
@GQuirino I believe we can extract this method in its own class/module and create a color dictionary that would avoid process the same color twice, I can help you to design that solution.

Reply:
We could also delegate `color_for` to that new class/module, It would be better testable and maintainable.